glossary.md

```diff
@@ -24,28 +40,90 @@ Zettle Go API Glossary
   <dd>A Zettle merchant account. It's used to hold funds that are ready to be paid out to merchants. After a transaction happens, the funds are recorded in the preliminary account while Zettle is checking whether the funds should be paid out. After that, the funds will be moved to and recorded in the liquid account while waiting to be paid out.</dd>
 </dl>
 
+<dl>
+  <dt>Location</dt>
+  <dd>The Inventory service keeps track of inventory balances by moving products between locations. These are created automatically when tracking is enabled for a product. A location can be a physical "Store", or virtual like "Sold", "Bin" (discarded products), or "Supplier" (for stock replenishment).</dd>
+</dl>
+
+<dl>
+  <dt>Low stock threshold</dt>
+  <dd>This is the minimum amount of inventory a merchant wants to have on hand. You can set the low stock threshold value through the Inventory API to help merchants manage their inventory.</dd>
+</dl>
+
 ## M
 <dl>
   <dt>Minor currency unit</dt>
   <dd>Also known as minor unit. A minor currency unit is the smallest unit of a currency. For example, for two-decimal currencies like US dollars, 199 with currency USD means $1.99 at Zettle. For zero-decimal currencies like Japanese Yen, 1095 with currency JPY means ¥1095 at Zettle.</dd>
 </dl>
 
 ## P
+<dl>
+  <dt>Partial return</dt>
+  <dd>Means that goods that is part of an original sale is being returned. In such a case, an amount is normally resolved and refunded from the original payment. See Refund.</dd>
```

Comment:
General comment - it may be more readable with full sentences. Fpr example, A partial refund means ...

Reply:
Fixed